.github/workflows/collector-slim.yml

```diff
@@ -67,41 +92,120 @@ jobs:
+          docker push "quay.io/stackrox-io/collector-builder:${COLLECTOR_BUILDER_TAG}-${{ matrix.arch }}"
 
       - name: Retag and push rhacs-eng -slim
         uses: ./.github/actions/retag-and-push
         with:
           src-image: ${{ inputs.collector-image }}
-          dst-image: ${{ env.RHACS_ENG_IMAGE }}-slim
+          dst-image: ${{ env.RHACS_ENG_IMAGE }}-${{ matrix.arch }}-slim
           username: ${{ secrets.QUAY_RHACS_ENG_RW_USERNAME }}
           password: ${{ secrets.QUAY_RHACS_ENG_RW_PASSWORD }}
 
       - name: Retag and push rhacs-eng -base
         uses: ./.github/actions/retag-and-push
         with:
           src-image: ${{ inputs.collector-image }}
-          dst-image: ${{ env.RHACS_ENG_IMAGE }}-base
+          dst-image: ${{ env.RHACS_ENG_IMAGE }}-${{ matrix.arch }}-base
           username: ${{ secrets.QUAY_RHACS_ENG_RW_USERNAME }}
           password: ${{ secrets.QUAY_RHACS_ENG_RW_PASSWORD }}
 
       - name: Push builder to rhacs-eng
-        if: github.event_name == 'push' || steps.pr-checks.outputs.builder-tag != env.DEFAULT_BUILDER_TAG
+        if: github.event_name == 'push' || needs.set-environment.outputs.collector-builder-tag != env.DEFAULT_BUILDER_TAG
         run: |
-          docker tag "quay.io/stackrox-io/collector-builder:${COLLECTOR_BUILDER_TAG}" "quay.io/rhacs-eng/collector-builder:${COLLECTOR_BUILDER_TAG}"
-          docker push "quay.io/rhacs-eng/collector-builder:${COLLECTOR_BUILDER_TAG}"
+          docker tag "quay.io/stackrox-io/collector-builder:${COLLECTOR_BUILDER_TAG}" "quay.io/rhacs-eng/collector-builder:${COLLECTOR_BUILDER_TAG}-${{ matrix.arch }}"
+          docker push "quay.io/rhacs-eng/collector-builder:${COLLECTOR_BUILDER_TAG}-${{ matrix.arch }}"
+
+  create-multiarch-manifest:
+    needs:
+    - set-environment
+    - build-collector-image
+    name: Create Multiarch manifest
+    runs-on: ubuntu-latest
+    env:
+      BUILD_BUILDER_IMAGE: ${{ needs.set-environment.outputs.build-builder-image }}
+      COLLECTOR_BUILDER_TAG: ${{ needs.set-environment.outputs.collector-builder-tag }}
+
+    steps:
+
+      - name: Login to quay.io/stackrox-io
+        uses: docker/login-action@v2
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAY_STACKROX_IO_RW_USERNAME }}
+          password: ${{ secrets.QUAY_STACKROX_IO_RW_PASSWORD }}  
+
+
+      - name: Create and push multiarch manifest for stackrox-io -slim
+        run: |
+          docker manifest create quay.io/stackrox-io/collector:${{ inputs.collector-tag }}-slim \
+                                 quay.io/stackrox-io/collector:${{ inputs.collector-tag }}-amd64-slim \
+                                 quay.io/stackrox-io/collector:${{ inputs.collector-tag }}-ppc64le-slim
+
+          docker manifest push quay.io/stackrox-io/collector:${{ inputs.collector-tag }}-slim
```

Comment:
[ultra-nit] We can probably refactor this into a small reusable action, since there are multiple blocks doing the same actions and only changing the image used. We can take care of it in a follow up PR though, not a blocker for this.

Reply:
Will fix in a follow up PR.